include/cudaq/Optimizer/Builder/Intrinsics.h

```diff
@@ -47,11 +47,14 @@ static constexpr const char getNumQubitsFromCudaqState[] =
     "__nvqpp_cudaq_state_numberOfQubits";
 
 // Create a new state from data.
-static constexpr const char createCudaqStateFromDataFP64[] =
+static constexpr const char createCudaqStateFromData64[] =
```

Comment:
curious: why get rid of the "fp" part of the name?

Reply:
I was trying to match the rest of the API but then realized that they are not consistent anyway:) I'll bring them back.

Reply:
reverted